troubleshooting/troubleshooting-android-whitelist-not-enabled.md

```diff
@@ -0,0 +1,62 @@
+---
+title: jQuery/AJAX/JSON Requests do not Work on Android
+description: jQuery API does not return any data, never fires on a real Android device.
+type: troubleshooting
+page_title: Internet Requests are Broken on Android.
+slug: android-whitelist-not-enabled
+position: 
+tags: appbuilder, android, 3rd-party-js-library, core-plugin, 
+teampulseid: 
+ticketid: 1065764, 1070518
+pitsid:
+---
+
+## Environment
+<table>
+  <tr>
+    <td>Service</td>
+    <td>{{site.ab-s}} ({{site.ab}})</td>	
+  </tr>
+  <tr>
+    <td>Plugin</td>
+    <td>Whitelist</td>	
+  </tr>
+  <tr>
+    <td>Mobile development type</td>
+    <td>Hybrid ({{site.ac}} app)</td>	
+  </tr>
+  <tr>
+    <td>Mobile OS</td>
+    <td>Android</td>	
+  </tr>
+  <tr>
+    <td>{{site.ac}} framework version</td>
+    <td>{{site.ac}} 4.0.0 and later</td>	
+  </tr>
+</table>
+
+## Description
+
+Hybrid application that is deployed on an Android device cannot do any internet requests.
+
+## Error Message
+
+Failed to load resource: the server responded with a status of 404 (Not Found) 
+
+## Cause
+
+{{site.ac}} 4.0 and above incorporates the {{site.ac}} whitelisting system, which provides a configurable security policy. For android, this policy is implemented by the [{{site.ac}} Whitelist Plugin](https://cordova.apache.org/docs/en/latest/reference/cordova-plugin-whitelist/). If the plugin is not enabled in the project, any internet request will respond with a status of 404 (Not Found).
+
+## Solution
+
+To enable the {{site.ac}} Whitelist Plugin and allow the internet request, complete the following steps:
+
+1. Open project **Properties**.
+1. Navigate to **Plugins** tab.
+1. Navigate to the last one of the **Core Plugins** section, which is **Whitelist**
+1. If the plugin switch is **OFF**, click on it to enable the plugin for the project.
```

Comment:
Mist/Proton Core Plugins do not have ON/OFF switch. You might consider using enabled/disabled.